packs/vsphere/actions/vm_hw_barebones_create.yaml

```diff
@@ -0,0 +1,52 @@
+---
+  name: "vm_hw_barebones_create"
+  runner_type: "run-python"
+  description: "Create BareBones VM (CPU, Ram, Graphics Only)"
+  enabled: true
+  entry_point: "vm_hw_barebones_create.py"
+  parameters:
+    vm_name:
+      type: "string"
+      description: "Name of the new Virtual Machine"
+      required: true
+    cluster:
+      type: "string"
+      description: "Cluster Name to Buid VM within"
+      required: true
+    datastore_cluster:
+      type: "string"
+      description: "Datastore Cluster for VM Files"
+      required: false
+    datastore:
+      type: "string"
+      description: "Datastore for VM Files"
+      required: false
+    description:
+      type: "string"
+      description: "Annotation text for Server"
+      required: false
+    resourcepool:
+      type: "string"
+      description: "Resource Pool for VM"
+      required: false
+    cpu_size:
+      type: "integer"
+      description: "Number of CPUs"
+      required: false
+      default: 1
+    ram_size:
+      type: "integer"
+      description: "Ammount of RAM in GB"
+      required: false
+      default: 2
+    guestos:
+      type: "string"
```

Comment:
Yes cpu/ram/guestos could be enums, but that does also mean i'm limiting a input.
cpu/ram can be as high as vsphere/hardware can offer which may vary setup to setup.
guestos will change with each release of vsphere.